src/main/java/me/ryanhamshire/GriefPrevention/BlockEventHandler.java

```diff
@@ -570,7 +575,7 @@ private void onPistonEvent(BlockPistonEvent event, List<Block> blocks)
         }
 
         // Pushing down or pulling up is safe if all blocks are in line with the piston.
-        if (minX == maxX && minZ == maxZ && direction == (event instanceof BlockPistonExtendEvent ? BlockFace.DOWN : BlockFace.UP)) return;
+        if (minX == maxX && minZ == maxZ && direction == (isRetract ? BlockFace.UP : BlockFace.DOWN)) return;
```

Comment:
idk if it was always like this but I rather have this notation expanded for readability - but I'll let it slide for now.